src/cli/tc.rs

```diff
@@ -12,39 +10,53 @@ use crate::config::Config;
 use crate::http::start_server;
 use crate::print_schema;
 
+struct StartCommand {
+  file_path: String,
+}
+
+struct CheckCommand {
+  file_path: String,
+  n_plus_one_queries: bool,
+  schema: bool,
+}
```

Comment:
We already have Command::Start and Command::Check
Not sure what we are doing with this additional types.

Reply:
I have resolved this issue and pushed. I have reused Command::Start and Command::Check to avoid creating new types. Please review my code and let me know if you have any comments or suggestions. Thank you.